library/aes.c

```diff
@@ -725,7 +725,7 @@ int mbedtls_aes_setkey_enc( mbedtls_aes_context *ctx, const unsigned char *key,
         return( mbedtls_aesni_setkey_enc( (unsigned char *) ctx->rk, key, keybits ) );
 #endif
 
-    mbedtls_platform_memset( RK, 0, ( keybits >> 5 ) * 4 );
+    mbedtls_platform_memset( RK, 0, keybits >> 3 );
```

Comment:
I thought it is an optimization, but now I see that after changing this code does not work in the same way. Could you explain why you made this change?

Reply:
This is neither an optimization, nor a behavior change - it's just about readability. 
The code works in the same way - what do you exactly mean?

Reply:
`( ( 124 >> 5 ) << 2 )` is not equivalent to `( 124 >> 3 )`. The First one is equal to 12 and the second is 15. The key here is the rest of the division

Reply:
The acceptable `keybits` values are checked in lines 695 - 710 and may only be one of three, either 128, 192, or 256. The function flow won't be able to reach this place with a different value, and for these values, the behavior is the same.

Reply:
If you want to leave it, could you add a comment that the least significant bits are always set to 0. I think it is important to understand this in the future.

Reply:
Done.